test/inductor/test_compiled_autograd.py

```diff
@@ -262,6 +270,107 @@ def fn():
         self.check_output_and_recompiles(fn, count=1)
 
 
+testdir = Path(__file__).absolute().parent.parent
+with mock.patch("sys.path", [*sys.path, str(testdir)]):
+    test_autograd = SourceFileLoader(
```

Comment:
nit: since I'm sure we're going to add more later, we'll most likely want to refactor that into being generic for other autograd related tests.